src/index.html

```diff
@@ -9,10 +9,75 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendars-container">
+      <div class="calendar calendar--start-day-sun calendar--month-lenght-31">
```

Comment:
There is a typo in the class name 'calendar--month-lenght-31'. It should be 'calendar--month-length-31'.